test/sensor_simulator/sensor_simulator.cpp

```diff
@@ -6,26 +6,34 @@ _ekf{ekf},
 _imu{ekf},
 _mag{ekf},
 _baro{ekf},
-_gps{ekf}
+_gps{ekf},
```

Comment:
I didn't catch this before, but you should be initializing these with
```suggestion
_gps(ekf),
```
since they all have the default constructor that takes the EKF.